test/textlint-rule-no-synonyms.test.ts

```diff
@@ -31,6 +31,13 @@ tester.run("textlint-rule-no-synonyms", rule, {
             options: {
                 preferWords: ["ユーザー"]
             }
+        },
+        // allowLexeme
+        {
+            text: "部屋の英語はroomです",
+            options: {
+                allowLexeme: false
```

Comment:
Is this `allowLexeme: true`?

Reply:
I wanted to test it for the case of `allowLexeme=false` and `allowAlphabet: true`. I updated the test to clarify what to be tested.